test/renaissance_web/controllers/login_controller_test.exs

```diff
@@ -1,17 +1,19 @@
 defmodule RenaissanceWeb.LoginControllerTest do
   use RenaissanceWeb.ConnCase
 
+  @user_params %{email: "mail@mail.com", password: "password"}
+
   test "GET /login" do
     conn = get(build_conn(), "/login")
 
     assert html_response(conn, 200) =~ "Login"
   end
 
   test "POST /login with correct email and password" do
-    post(build_conn(), "/register", %{"user" => %{email: "mail@mail.com", password: "password"}})
-
     conn =
-      post(build_conn(), "/login", %{"user" => %{email: "mail@mail.com", password: "password"}})
+      build_conn()
+      |> post("/register", %{"user" => @user_params})
```

Comment:
Rather than relying on using the controllers to put the connection in the expected state (signed in), user a setup block to just create a user and assign to the session, returning a signed in connection.